lib/letters.mli

```diff
@@ -84,14 +86,26 @@ type recipient =
     or HTML message
 
     Returns [result] indicating if built email is valid or Error if anything failed *)
+val create_email
+  :  ?reply_to:string
+  -> from:string
+  -> recipients:recipient list
+  -> subject:string
+  -> body:body
+  -> unit
+  -> (Mrmime.Mt.t, string) result
+
+(** Build an email using mrmime like [create_email]
+
+    Backwards compatible build function without reply_to *)
 val build_email
```

Comment:
@mabiede Maybe adding `[@@deprecated ...]` could make sense, so we can rid of it in a later release?

Reply:
@joseferben added deprecation warnings and updated the documentation